vaadin-testbench-shared/pom.xml

```diff
@@ -0,0 +1,232 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-testbench-parent</artifactId>
+        <version>9.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>vaadin-testbench-shared</artifactId>
+    <packaging>jar</packaging>
+    <name>Vaadin TestBench Shared</name>
```

Comment:
Please add a `<description>`, telling a couple of word about this module: Vaadin TestBench shared classes not dependant on any concrete testing framework.